debian/postgresql-metrics.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=postgresql-metrics process
+StartLimitInterval=100
+StartLimitBurst=10
+
+[Service]
+Type=forking
```

Comment:
I'm not sure if the type should be simple here. Postgresql-metrics doesn't fork or run as daemon. Cannot tell top of my head what the difference would be though. Will check later unless you can elaborate?

Reply:
Good catch! It doesn't make a lot of sense to use the forking type if it doesn't fork. I'll change it to simple.